### PR TITLE
fix: key interaction handling no longer prevents "tab" presses

### LINKED ISSUE #1692: Tab Order Gets Trapped in Color Elements
### Expected Behaviour
When I tab through an interface built with `<sp-color-area>` and `<sp-color-slider>` I am able to visit AND leave each element as I continue through the tab order.

### Actual Behaviour
The tab key fails to leave the `<sp-color-area>` and `<sp-color-slider>` once they've acquired forcus.

### Reproduce Scenario (including but not limited to)
Vist: https://webcomponents.dev/edit/uY4MiHRPIvWaix4CXFNk/src/index.stories.js
1. Tab into the interface (focuses "Solid" Action Menu)
2. Next tab (focuses "ellipse" menu)
3. Next tab (focuses color wheel)
4. Next tab (focuses color area)
5. Next tab does not deliver visible change...focus also unchanged.

OR:

1. Focus opacity text input.
2. Shift + tab to focus color slider (handle enlarges)
3. Tab or shift + tab does not deliver visible change...focus also unchanged. 

#### Steps to Reproduce
- screen cap TK

#### Possible causes...
- it appears that the preventDefault call to override the interface with the underlying `<input/>` is called on all keystrokes and not only on those expected to alter the value of the control.

#### Platform and Version
macOS/Chrome 92

#### Sample Code that illustrates the problem
https://webcomponents.dev/edit/uY4MiHRPIvWaix4CXFNk/src/index.stories.js

#### Logs taken while reproducing problem
None.

### PR DESCRIPTION
**Pull request by Westbrook**
## Description
- ensure the `<sp-color-area>` and `<sp-color-sldier>` have a single tab stop and the the tab order does not get trapped within these elements

## Related Issue
fixes #1692 

## Motivation and Context
Color application interfaces should be keyboard accessible.

## How Has This Been Tested?
- unit tests
- manual test

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by Westbrook** [comment]
Your expectation is correct! Nice catch. This would definitely count as a separate bug to correct. I can take a look at where that would fit in here, or we can create a new issue separately to track...

**Review by hunterloftis — APPROVED** [review]
lgtm. I was able to tab into and out of the area, slider, and wheel.

File: packages/color-area/src/ColorArea.ts [hunk 1]
```diff
@@ -211,15 +212,15 @@ export class ColorArea extends SpectrumElement {
...
         const { key, code } = event;
-        event.preventDefault();
         if (['Shift', 'Meta', 'Control', 'Alt'].includes(key)) {
             this.altKeys.add(key);
             this.altered = this.altKeys.size;
         }
         if (code.search('Arrow') === 0) {
             this.activeKeys.add(code);
+            event.preventDefault();
+            this.handleKeypress();
```
[the remark this answers is not in the document]
Comment (Westbrook) [reply packages/color-area/src/ColorArea.ts:222]: "Up", "Down", "Left", and "Right" comes from referencing the W3C Aria Specs too often. They include this in all of their example code. I looked it up at one point, but I can neither remember or find what browser that supported (an old one) right now. Being it was already gated in the code path, I've removed them. 

The ideas around DRY are good, and will make a reappearance when we discuss what part of the color elements could be shared in future versions. Skipping for now.

[the remark this answers is not in the document]
Comment (hunterloftis) [reply packages/color-area/src/ColorArea.ts:222]: sgtm, I'll re-approve

**Review by hunterloftis — APPROVED** [review]

**Pull request merged**